<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FeatureExp</class>
 <widget class="QWidget" name="FeatureExp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>478</width>
    <height>237</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>特征导出</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>参数设置</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="4" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>数据库名:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>预训练网络:</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>mini_batches:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>预导出的层:</string>
        </property>
       </widget>
      </item>
      <item row="7" column="4">
       <widget class="QPushButton" name="pushButton_DoExp">
        <property name="text">
         <string>执行</string>
        </property>
       </widget>
      </item>
      <item row="3" column="4">
       <widget class="QPushButton" name="pushButton_BrowserPath">
        <property name="text">
         <string>浏览</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>网络结构部署描述:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="4">
       <widget class="QPushButton" name="pushButton_Verify">
        <property name="text">
         <string>验证</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>导出位置:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QPushButton" name="pushButton_BrowserModel">
        <property name="text">
         <string>浏览</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QPushButton" name="pushButton_BrowserProto">
        <property name="text">
         <string>浏览</string>
        </property>
       </widget>
      </item>
      <item row="8" column="4">
       <widget class="QRadioButton" name="radioButton_leveldb">
        <property name="text">
         <string>leveldb</string>
        </property>
       </widget>
      </item>
      <item row="8" column="3">
       <widget class="QRadioButton" name="radioButton_lmdb">
        <property name="text">
         <string>lmdb</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1" colspan="2">
       <widget class="QComboBox" name="comboBox_cupgpu">
        <item>
         <property name="text">
          <string>CPU</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>GPU</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="7" column="1" colspan="3">
       <widget class="QLineEdit" name="lineEdit_MiniBatches"/>
      </item>
      <item row="5" column="1" colspan="3">
       <widget class="QLineEdit" name="lineEdit_LayerID"/>
      </item>
      <item row="4" column="1" colspan="3">
       <widget class="QLineEdit" name="lineEdit_DbName"/>
      </item>
      <item row="3" column="1" colspan="3">
       <widget class="QLineEdit" name="lineEdit_ExpPath"/>
      </item>
      <item row="2" column="1" colspan="3">
       <widget class="QLineEdit" name="lineEdit_Prototxt"/>
      </item>
      <item row="0" column="1" colspan="3">
       <widget class="QLineEdit" name="lineEdit_Model"/>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>运行选项:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_BrowserModel</sender>
   <signal>clicked()</signal>
   <receiver>FeatureExp</receiver>
   <slot>Browser1()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1022</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>715</x>
     <y>39</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_BrowserProto</sender>
   <signal>clicked()</signal>
   <receiver>FeatureExp</receiver>
   <slot>Browser2()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1010</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>715</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_BrowserPath</sender>
   <signal>clicked()</signal>
   <receiver>FeatureExp</receiver>
   <slot>Browser3()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1014</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>716</x>
     <y>112</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Verify</sender>
   <signal>clicked()</signal>
   <receiver>FeatureExp</receiver>
   <slot>Verification()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>995</x>
     <y>154</y>
    </hint>
    <hint type="destinationlabel">
     <x>715</x>
     <y>156</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_DoExp</sender>
   <signal>clicked()</signal>
   <receiver>FeatureExp</receiver>
   <slot>DoFeatureExp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1007</x>
     <y>188</y>
    </hint>
    <hint type="destinationlabel">
     <x>715</x>
     <y>202</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>Browser1()</slot>
  <slot>Browser2()</slot>
  <slot>Browser3()</slot>
  <slot>Verification()</slot>
  <slot>DoFeatureExp()</slot>
 </slots>
</ui>
